plugin/trino-kafka/pom.xml

```diff
@@ -79,11 +79,21 @@
             <artifactId>guice</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>kafka-avro-serializer</artifactId>
```

Comment:
What is the license of this?

Reply:
Apache license. Same applies to the kafka-schema-registry-client artifact.